src/com/metacube/get2019/shop.java

```diff
@@ -0,0 +1,250 @@
+package com.metacube.get2019;
+
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+class Item
+{
+int id;
+String itemName;
+double price;
+
+/*
+* This method is used to set values of 3 properties and to print them.
+* @param id This is first parameter to set id of item
+* @param itemName this is second parameter to set itemname
+* @param price this is third parameter to set price of item
+*/
+public Item(int id, String itemName, double price){
+this.id=id;
+this.itemName=itemName;
+this.price=price;
+System.out.println(id+"\t\t"+itemName+"\t\t"+price);
+}
+
+/*
+* This method is used to return value of itemname
+* @return string type itemname
+*/
+public String getItemName(){
+return itemName;
+}
+
+/*
+* This method is used to return value of id.
+* @return id of item
+*/
+public int getid(){
+return id;
+}
+
+/*
+* This method is used to return the price of the item.
+* @return price of item
+*/
+public double getprice(){
+return price;
+}
+}
+
+/*
+ * This class has the attributes pro(Product),quant(Quantity), 
+ * pid(ProductID) and price(Cost) 
+ * This class also has all the functions of a Shopping Cart
+ */
+class ShoppingCart
+{
+	//ArrayList are for storing the input
+	ArrayList<String> pro=new ArrayList<String>();
+	ArrayList<Integer> quant=new ArrayList<Integer>();

```

Comment:
Isn't there a better way to deal with the cartItem instead of defining arrays globally?

Reply:
added for the future extensibility